public/locales/uk/translation.json

```diff
@@ -552,7 +554,7 @@
         "showfps": "Показувати кількість кадрів (DX9, 10 та 11)",
         "start-in-tray": "Запускати згорнутим",
         "steamruntime": "Використовувати Steam Runtime",
-        "winecrossoverbottle": "Пляшка CrossOver",
+        "winecrossoverbottle": "CrossOver Bottle",
```

Comment:
should this be translated @flavioislima? since it is the product name

Reply:
I think Crossover is the name that should not be translated. Bottle is fine.